src/SourceBuild/content/eng/pipelines/source-build-sdk-diff-tests.yml

```diff
@@ -1,6 +1,6 @@
 schedules:
-- cron: "0 11 * * 1-5"
-  displayName: Run on weekdays at 11am UTC
+- cron: "0 12 * * 1-5"
+  displayName: Run on weekdays at 12pm UTC

```

Comment:
Is this enough of a buffer in case the build runs long?  Consider increasing the spread by 1 more hour (by starting the VMR build 1 hour earlier).

Reply:
The VMR pipeline schedule is defined in AzDO UI. I've set that to be an hour earlier.